crates/pathfinder/src/lib.rs

```diff
@@ -11,3 +11,15 @@ pub mod sequencer;
 pub mod state;
 pub mod storage;
 pub mod update;
+
+macro_rules! starkhash {
+    ($hex:expr) => {{
+        let bytes = hex_literal::hex!($hex);
+        match stark_hash::StarkHash::from_be_slice(bytes.as_slice()) {
+            Ok(sh) => sh,
+            Err(stark_hash::OverflowError) => panic!("Invalid constant: OverflowError"),
+        }
+    }};
+}
+
```

Comment:
```suggestion
/// Creates a [StarkHash] from a hex string, panics if it is invalid.
macro_rules! starkhash {
    ($hex:expr) => {{
        let bytes = hex_literal::hex!($hex);
        match stark_hash::StarkHash::from_be_slice(bytes.as_slice()) {
            Ok(sh) => sh,
            Err(stark_hash::OverflowError) => panic!("Invalid constant: OverflowError"),
        }
    }};
}
```
or something similar.

Reply:
Done.